resources/js/views/settings/servers/View.vue

```diff
@@ -0,0 +1,337 @@
+            this.online = 0;
+            this.offlineReason = 'salt';
+          } else {
+            this.online = 0;
+            this.offlineReason = 'connection';
+          }
+        }
+      }).catch(error => {
+        this.online = null;
+        this.offlineReason = null;
+        Base.error(error, this.$root, error.message);
+      }).finally(() => {
+        this.checking = false;
+      });
+    },
+
+    /**
+     * Loads the servers from the backend
+     */
+    load () {
+      this.modelLoadingError = false;
+
+      if (this.id !== 'new') {
+        this.isBusy = true;
+
+        Base.call(`servers/${this.id}`).then(response => {
+          this.model = response.data.data;
+          this.model.disabled = this.model.status === -1;
+          this.online = this.model.status === -1 ? null : this.model.status;
+          this.offlineReason = null;
+        }).catch(error => {
+          if (error.response && error.response.status === env.HTTP_NOT_FOUND) {
+            this.$router.push({ name: 'settings.servers' });
+          } else {
+            this.modelLoadingError = true;
+          }
+          Base.error(error, this.$root, error.message);
+        }).finally(() => {
+          this.isBusy = false;
+        });
+      }
+    },
+
+    /**
+     * Saves the changes of the server to the database by making a api call.
+     *
+     * @param evt
+     */
+    saveServer (evt) {
+      if (evt) {
+        evt.preventDefault();
+      }
+
+      this.isBusy = true;
+
+      const config = {
+        method: this.id === 'new' ? 'post' : 'put',
+        data: this.model
+      };
+
+      Base.call(this.id === 'new' ? 'servers' : `servers/${this.id}`, config).then(() => {
+        this.$router.push({ name: 'settings.servers' });
+      }).catch(error => {
+        if (error.response && error.response.status === env.HTTP_UNPROCESSABLE_ENTITY) {
+          this.errors = error.response.data.errors;
+        } else if (error.response && error.response.status === env.HTTP_STALE_MODEL) {
+          this.staleError = error.response.data;
+          this.$refs['stale-server-modal'].show();
+        } else {
+          if (error.response && error.response.status === env.HTTP_NOT_FOUND) {
+            this.$router.push({ name: 'settings.servers' });
+          }
+
+          Base.error(error, this.$root, error.message);
+        }
+      }).finally(() => {
+        this.isBusy = false;
+      });
+    },
+
+    /**
+     * Force a overwrite of the server in the database by setting the `updated_at` field to the new one.
+     */
+    forceOverwrite () {
+      this.model.updated_at = this.staleError.new_model.updated_at;
+      this.staleError = {};
+      this.$refs['stale-server-modal'].hide();
+      this.saveServer();
+    },
+
+    /**
+     * Refreshes the current model with the new passed from the stale error response.
+     */
+    refreshServer () {
+      this.model = this.staleError.new_model;
```

Comment:
During the refresh the api details gets lost. This is caused by the resource `withApi` method in the backend since this not gets called in the `EnsureModelNotStale` middleware.

Reply:
Good catch, it also happens on the user view. I'll work on a fix